pkg/config/agent/dynamic/adminqos/reclaimedresource/reclaimedresource_base.go

```diff
@@ -72,8 +75,50 @@ func (c *ReclaimedResourceConfiguration) ApplyConfiguration(conf *crd.DynamicCon
 				c.MinReclaimedResourceForAllocate[resourceName] = value
 			}
 		}
+
+		if config.MaxNodeUtilizationPercent != nil {
+			for resourceName, value := range config.MaxNodeUtilizationPercent {
+				c.MaxNodeUtilizationPercent[resourceName] = value
+			}
+		}
 	}
 
 	c.CPUHeadroomConfiguration.ApplyConfiguration(conf)
 	c.MemoryHeadroomConfiguration.ApplyConfiguration(conf)
 }
+
+func (c *ReclaimedResourceConfiguration) GetReservedResourceForAllocate(nodeResourceList v1.ResourceList) v1.ResourceList {
```

Comment:
It may be better for GetReservedResourceForAllocate to use resource name and quantity as two separate input parameters.